pkg/operator/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -69,8 +71,14 @@ func syncOpenShiftAPIServer_v311_00_to_latest(c OpenShiftAPIServerOperator, orig
 		errors = append(errors, fmt.Errorf("%q: %v", "client-ca", err))
 	}
 
+	imageImportCAModified := false
```

Comment:
why not just `imageImportCAModified, err :=` ?

Reply:
leftover from how i iterated my way there. will fix.